CLAUDE.md

```
@@ -1,8 +1,16 @@
 # Gmail MCP Server - Development Guidelines
 
-## Build/Test/Run Commands
-- **Build**: `cargo build`
-- **Run**: `cargo run`
+## Gmail MCP Server (rust-mcp-sdk)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)